### ISSUE TITLE
Errors and warnings not read out by screen reader when they appear

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#93014]
Issue Type: <b>Bug</b>

1. Activate orca.
2. activate vscode.
3. open settings.
4. Include in the file the following line:
> "window.titleBarStyle": "custom"
5. Save the file.

No alert is issued by the orca stating that there is an error in the file. However, if I access the problem area by pressing ctrl + shift + m I have access to the error.
It seems to me that it would be interesting if there was some information about the existence of errors at some point.

@isidorn  Does the sign exist but the orca just isn't reading?

VS Code version: Code - Insiders 1.44.0-insider (c0eea2a712471a4d242d15704cfa968241d63280, 2020-03-19T09:28:15.202Z)
OS version: Linux x64 5.5.9-arch1-2

**Comment by isidorn** [comment]
This is a general problem that we seem to have that problems are only read out once you focus on the problems view.
When would you expect to hear the problem? As soon as it happens or when you are closing the file?
Since we can only convey problems via `aria.alerts` and I am worried if this does not end up being too spammy for our users if we use an `aria.alert` every time there is an error.

**Comment by jvesouza** [comment]
> When would you expect to hear the problem? As soon as it happens or when you are closing the file?

When the file is saved, although the auto save option can generate some noise.

**Comment by isidorn** [comment]
@jvesouza I like that idea. So when you explicilty save the file we should use `aria.alert` and announce all the errors and warnings in that file.
@sandy081 is it possible that the markers service does this, should be doable?

**Comment by jvesouza** [comment]
> @jvesouza I like that idea. So when you explicilty save the file we should use `aria.alert` and announce all the errors and warnings in that file.
In my opinion it would not be necessary to announce all errors. Suffice it to inform that there are errors / warnings.
Knowing that there are errors / warnings we can go to the problem view to identify them.

**Comment by sandy081** [comment]
Yeah can be do-able. But need to hook this with save operation.

**Comment by isidorn** [comment]
@sandy081 great
@jvesouza yeah we can do a short alert "There are errors / warnings in file 'hello.txt'"

**Comment by jvesouza** [comment]
@isidorn  Now that I can access the contents of the status bar quite simply, I think that this feature would not need to be implemente. 

I always have the habit of checking the status bar and information about the existence of errors or warnings is available there.

**Comment by isidorn** [comment]
@jvesouza I like your point. Since this is in the status bar now we are not in a bad spot.
So let's close for now and we can reopen in the future if more users request this. Thanks
